spec/message_handler_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'pg'
+require_relative '../lib/message_handler'
+
+describe MessageHandler do
+  describe '.all' do 
+    let(:db_entry) { ["id" => "1047", "messages" => "Hello World", "timestamp" => "01/04/2022"] }
+    let(:db_double) { double(:dbdouble, add: db_entry, all: db_entry) }
```

Comment:
I like how you've made testing with mocks possible by having a separate database class.